surveys/2024-annual-survey/translations/de.md

```diff
@@ -163,7 +163,7 @@ Type: select all that apply
 - Zur Nutzung eines oder mehrerer Sprachfeatures die ich brauche
 - Ich mag es, Zugriff auf die neuesten Features zu haben
 - Um zu helfen, die Nightly Version auf Bugs zu testen
-- Um Design-Feedback zu nächtlichen Funktionen zu geben
+- Um Design-Feedback zu Nightly Funktionen zu geben
```

Comment:
For consistency keep the English term here instead of translating "nightly"; this is also consistent with usage in previous questions a few lines above

Reply:
The deciding factor should be which is the most common used term by the Rust German-speaking community. What would your experience in the German Rust communities suggest?

FWIW this unofficial translation of the Rust book[^0] says "nachtlich", so I am inclined to think it is commonly accepted? But I can't really say.

[^0]: https://rust-lang-de.github.io/rustbook-de/ch00-00-introduction.html

Reply:
I am not active in the German Rust community, sorry.

Though I guess the term "nightly" is likely familiar to German users, since it occurs in compiler errors and also in Cargo commands.

In the German translation of the old Rust website it was called "Nightly" as well: https://github.com/rust-lang/prev.rust-lang.org/blob/master/de-DE/documentation.md
(side note: the new website has no official German translation)

Also, when restricting Google search results to German, it seems "rust" / "rust lang" "nightly" gives more results whereas "nächtlich" seems to already give results about the German city Rust and the video game Rust on the first result page.

Reply:
:man_shrugging: 

let's go with this